authors/apps/friends/views.py

```diff
@@ -34,17 +44,23 @@ def post(self, request, username, format=None):
                 {"message": "You cannot follow yourself"},
                 status=status.HTTP_406_NOT_ACCEPTABLE)
 
-        if Friend.objects.filter(user_from=follower, user_to=followed):
+        is_following = Friend.objects.get_or_create(
```

Comment:
While this works, it's more readable to assign different variables to the return value of a function that returns a tuple. Something like 
`user_followed, created_now = Friend.objects.get_or_create...`